protocol-units/da/movement/protocol/util/src/blob/ir/data.rs

```diff
@@ -27,11 +27,11 @@ where
 		Self { blob, timestamp, __curve_marker: std::marker::PhantomData }
 	}
 
-	pub fn now(blob: Vec<u8>) -> Result<Self, anyhow::Error> {
-		Ok(Self::new(
-			blob,
-			std::time::SystemTime::now().duration_since(std::time::UNIX_EPOCH)?.as_secs(),
-		))
+	pub fn now(blob: Vec<u8>) -> Self {
+		// mark the timestamp as now in milliseconds
+		let timestamp = chrono::Utc::now().timestamp_micros() as u64;
```

Comment:
I've changed the comment after [a review from @mzabaluev ](https://github.com/movementlabsxyz/movement/pull/1067/files#diff-73055257164a15172d8a5e008ac12f6291a0291e62da137ee0380c16f8905f0b).